frb/surveys/catalog_utils.py

```diff
@@ -165,6 +166,114 @@ def summarize_catalog(frbc, catalog, summary_radius, photom_column, magnitude):
     # Return
     return summary_list
 
+def _detect_mag_cols(photometry_table):
+    """
+    Searches the column names of a 
+    photometry table for columns with mags.
+    Args:
+        photometry_table: astropy Table
+            A table containing photometric
+            data from a catlog.
+    Returns:
+        mag_colnames: list
+            A list of column names with magnitudes
+        mag_err_colnames: list
+            A list of column names with errors
+            in the magnitudes.
+    """
+    assert type(photometry_table)==Table, "Photometry table must be an astropy Table instance."
+    allcols = np.array(photometry_table.colnames)
+
+    #Start with DES
+    photom_cols = []
+    photom_errcols = []
+
+    descols = [col for col in allcols if "DES_" in col and "tile" not in col and "ID" not in col and "err" not in col]
+    des_errcols = [col for col in allcols if "DES_" in col and "err" in col]
+    
+    photom_cols += descols
+    photom_errcols += des_errcols
+
+    #WISE
```

Comment:
Do you think there might be a way to do the following blocks of code
in a semi-automated fashion using the filters defined in `frb/galaxies/defs.py` ?

Reply:
It can be once all tables return WISE columns with the same column names uniformly. I'll implement that.